src/components/Login/Login.jsx

```diff
@@ -0,0 +1,112 @@
+import "../Signup/authentication.css";
+import axios from "axios";
+//import { getTestData, getCredentials } from "../../utils";
+import { useNavigate, Link, useLocation } from "react-router-dom";
+import { useState } from "react";
+
+
+export const Login = () => {
+  // const { setIsLoggedin, setUserData } = useAuth();
+  // const navigate = useNavigate();
+  // const location = useLocation();
+  const [error, setError] = useState(false);
+  //const { successToast, errorToast } = useToast();
+
+  const testLogin = async () => {
+    // try {
+    //   const response = await axios.post("/api/auth/login", getTestData());
+    //   if (response.data.encodedToken) {
+    //     localStorage.setItem(
+    //       "login",
+    //       JSON.stringify(response.data.encodedToken)
+    //     );
+    //     localStorage.setItem("user", JSON.stringify(response.data.foundUser));
+    //     setUserData(response.data.foundUser);
+    //     setIsLoggedin(true);
+    //     successToast("Welcome to FreshView Video Library");
+    //     navigate(location?.state?.from?.pathname || "/");
+    //   }
+    // } catch (e) {
+    //   errorToast("Some Error Occured...");
+    //   setError(true);
+    //   navigate("/login");
+    // }
+  };
+
+  const handleLogin = async (event) => {
+    // try {
+    //   event.preventDefault();
+    //   const { email, password } = event.target.elements;
+    //   const response = await axios.post(
+    //     "/api/auth/login",
+    //     getCredentials(email, password)
+    //   );
+    //   if (response.data.encodedToken) {
+    //     localStorage.setItem(
+    //       "login",
+    //       JSON.stringify(response.data.encodedToken)
+    //     );
+    //     setUserData(response.data.foundUser);
```

Comment:
Remove commented code

Reply:
This one required for next development so commented it...